fusillade-api.yml

```diff
@@ -234,7 +234,33 @@ paths:
         - oauth
       responses:
         200:
-          description: OK.
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  sub:
```

Comment:
At first glance, `sub` is not a descriptive name for a property (although I'm generally unfamiliar with what information should be expected in userinfo).

Reply:
This is a oauth2 standard for a unique user identifier or subject. It's out of my hands.